corehq/apps/domain/views/tombstone.py

```diff
@@ -78,23 +76,20 @@ class TombstoneManagementForm(forms.Form):
         widget=forms.Textarea()
     )
 
+    @staticmethod
+    def split_csv(comma_separated_list):
+        return list(
+            filter(None, (domain.strip() for domain in comma_separated_list.split(','))))
+
     def clean(self):
         csv_domain_list = self.cleaned_data.get('csv_domain_list', '')
-        domains = csv_domain_list.split(',')
-
-        if len(domains) > 10:
-            raise forms.ValidationError(
-                "This command is intended to create a few tombstones at a time. "
-                "If you need to create more than 10, do them in batches."
-            )
-
-        self.cleaned_data['domains'] = domains
+        self.cleaned_data['domains'] = self.split_csv(csv_domain_list)
         return self.cleaned_data
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        self.helper = FormHelper()
+        self.helper = HQFormHelper()
```

Comment:
Sorry, wasn't clear earlier: this lets you delete the three lines below: https://github.com/dimagi/commcare-hq/blob/868148af1ef1c32dcadfca265866ad4bd53fa1e8/corehq/apps/hqwebapp/crispy.py#L22-L25

Reply:
Ah nice thanks